src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigure2OptionListener.java

```diff
@@ -17,5 +19,8 @@ public SelectTestFigure2OptionListener(DriverManager driverManager) {
 	@Override
 	public void actionPerformed(ActionEvent e) {
 		FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+		MacroCommandManager manager = MacroFeature.getDriverCommandManager();
+		FiguresJoe.figureScript2(manager);
+		manager.setCurrentCommand(manager.getCommands(),"FigureJoe2");
```

Comment:
Figure tests should not be changed at all! MACRO feature can be transparent, because macro recorder is a driver (it can be made available through driverManager.getCurrentDriver())